dbt_worker/tasks.py

```diff
@@ -113,6 +118,9 @@ def _invoke_runner(
             {"exc_type": type(e).__name__, "exc_message": str(e)},
             callback_url,
         )
+        # TODO: make this work
```

Comment:
:P

Reply:
😅 Yeah, this doesn't actually work bc of celery burying logs, but I went to the trouble of figuring out the new event interface so I figured I might as well keep it. The implementation should be right, just need to route celery logs to the root logger.